docs/querying/api.md

```diff
@@ -572,6 +572,11 @@ guarantees as the overarching API v1.
 ```
 GET /api/v1/rules
 ```
+```
+# Limit the number of alerts and series individual rules can produce.
+# 0 is no limit.
+[ limit: <int> | default = 0 ]
+```
```

Comment:
This snippet isn't needed here. It expresses the configuration option, but is different from the API response.